### PR TITLE
SDXL: Reconstruct conditioner class

### PR DESCRIPTION
**Pull request by zhanghuiyao**

### CODE REVIEW

**Review by SamitHuang — APPROVED** [review]
lgtm

File: examples/stable_diffusion_xl/gm/modules/embedders/modules.py [hunk 1]
```diff
@@ -142,16 +161,51 @@ def __call__(self, batch: Dict, force_zero_embeddings: Optional[List] = None) ->
...
+                        crossattn = ops.concat((crossattn, emb), 2)
+                else:  # concat
+                    if concat is None:
+                        concat = emb
+                    else:
+                        concat = ops.concat((concat, emb), 1)
+
+        return vector, crossattn, concat
+
+    def __call__(self, batch: Dict, force_zero_embeddings: Optional[List] = None) -> Dict:
```
[the remark this answers is not in the document]
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/gm/modules/embedders/modules.py:187]: 不会冲突 graph默认走construct，pynative走__call__，兼容原推理流程

File: examples/stable_diffusion_xl/gm/modules/embedders/open_clip/transformer.py [hunk 2]
```diff
@@ -158,7 +212,8 @@ def attention(
         if attn_mask is not None:
             attn_mask = attn_mask.astype(q_x.dtype)
 
-        return self.attn(q_x, k_x, v_x, need_weights=False, attn_mask=attn_mask)[0]
+        # return self.attn(q_x, k_x, v_x, need_weights=False, attn_mask=attn_mask)[0]
```
[the remark this answers is not in the document]
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/gm/modules/embedders/open_clip/transformer.py]: done

File: examples/stable_diffusion_xl/gm/modules/embedders/open_clip/transformer.py [hunk 3]
```diff
@@ -124,7 +177,8 @@ def __init__(
         super().__init__()
 
         self.ln_1 = norm_layer([d_model])
-        self.attn = nn.MultiheadAttention(d_model, n_head)  # .to_float(ms.float16)
+        # self.attn = nn.MultiheadAttention(d_model, n_head)  # .to_float(ms.float16)
```
[the remark this answers is not in the document]
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/gm/modules/embedders/open_clip/transformer.py]: done

File: examples/stable_diffusion_xl/gm/modules/attention.py [hunk 4]
```diff
@@ -8,6 +8,7 @@
 from mindspore import nn, ops
 
 try:
+    # from mindspore.nn.layer.flash_attention import FlashAttention
```
[the remark this answers is not in the document]
Comment (zhanghuiyao) [reply examples/stable_diffusion_xl/gm/modules/attention.py:12]: done

**Review by CaitinZhao — APPROVED** [review]

**Pull request merged**